# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: editor/libeditor/EditorUtils.cpp
# Commit: 71670cedaf26
# Full Hash: 71670cedaf26214f3bcb8562a445cf1ef24f4db6
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2021-06-30 09:36:48
# Regressor Bug: 1714914
# File Overlap Count: 1
# Description:
#   Bug 1714914 - Make editor classes treat offset in parent node as uint32_t r=m_kato
#   
#   It should be treated as `uint32_t` since DOM API does so.  However, there are
#   some exceptions:
#   
# ==============================================================================

diff -r f5dcc2d793d6 -r 71670cedaf26 editor/libeditor/EditorUtils.cpp
--- a/editor/libeditor/EditorUtils.cpp	Tue Jun 29 22:00:09 2021 +0000
+++ b/editor/libeditor/EditorUtils.cpp	Wed Jun 30 07:07:28 2021 +0000
@@ -560,7 +560,7 @@
 
 bool EditorUtils::IsPointInSelection(const Selection& aSelection,
                                      const nsINode& aParentNode,
-                                     int32_t aOffset) {
+                                     uint32_t aOffset) {
   if (aSelection.IsCollapsed()) {
     return false;
   }